docs/mine-hnt/validators/mainnet/wallet.mdx

```diff
@@ -145,21 +145,22 @@ unstaked.
 ### Unstake Tokens
 
 If you unstake tokens the validator node enters a stake cooldown period. During this cooldown period
-the validator node will not earn rewards and at the end of the cooldown period (currently 250,000
-blocks or approximately 5.7 months in mainnet) your initial staking amount will be released to the
+the validator node will not earn rewards and at the end of the cooldown period (currently 125,000
+blocks or approximately 3 months in mainnet) your initial staking amount will be released to the
 wallet.
 
 To ensure the operator understands this unstake operation is not reversible and cannot be shortened,
 there are additional calculations to be performed in the unstake command. You need to include the
 stake release block height which should be at least the current block height plus the cooldown
 period, and 5-10 blocks to allow for chain processing delays.
 
-For example, if the current block height was 5000, you'd add 250,000 for the stake withdrawl
+For example, if the current block height was 1,537,797, you'd add 125,000 for the stake withdrawl
 cooldown period, plus an additional 10 blocks to allow for chain processing delays. Therefore the
-`stake-release-height` = `5000 + 250000 + 10` or `255010`.
+`stake-release-height` = `1537797 + 125000 + 10` = `1662807`.
```

Comment:
maybe add a note to be explicit about the current block height. People like to copy paste here and fail. They end up coming into #validator-ops with this question.